src/pages/fr/getting-started.md

```diff
@@ -2,107 +2,108 @@
 
 ## Apprendre Astro
 
-Voici quelques exemples de concepts et modèles de sites construits avec Astro !
-
-📚 En savoir plus sur la [Structure d'un Projet](/fr/core-concepts/project-structure/) Astro.
+Voir des exemples de certains des concepts et modèles clés d'un site Astro !
 
-📚 Apprendre plus à propos des [Directives de Composants](/fr/reference/directives-reference/) inclus dans Astro.
+📚 [Ajouter une première page](/fr/core-concepts/astro-pages/) à votre site.
 
-📚 Explorez l'[API](/fr/reference/api-reference/) d'Astro.
+📚 En savoir plus sur la [structure de projet](/fr/core-concepts/project-structure/) d'Astro.
 
-*... et encore plus d'infos dans la catégorie **Référence***
+📚 En savoir plus sur le [routage basé sur les fichiers](/fr/core-concepts/routing/) d'Astro.
 
-## Étendre l'écosystème Astro
+*... retrouvez notre documentation d'API complète sous l'onglet **Référence**.*
 
-🧰 Démarrez votre prochain projet avec un [Thème Pré-construit](https://astro.build/themes/).
+## Etendre Astro
 
-🧰 Personnalisez votre site avec des [Plugins et Composants](https://astro.build/integrations/) officiels et communautaires.
+🧰 Démarrez votre prochain projet avec un [thème prédéfini](https://astro.build/themes/)
 
-🧰 Inspirez-vous en visitant notre [Présentation de Sites](https://astro.build/showcase/).
+🧰 Personnalisez votre site avec les [plugins et composants](https://astro.build/integrations/) officiels et ceux de la communauté.
 
-*... allez voir notre [Guide d'utilisation des Intégrations](/fr/guides/integrations-guide/)*
+🧰 Soyez inspiré en visitant le [showcase de sites](https://astro.build/showcase/).
 
-## Rejoindre notre communauté
+*... voir notre [guide sur l'utilisation des intégrations](/fr/guides/integrations-guide/)*.
 
-Rejoignez [le Discord d'Astro](https://astro.build/chat/) pour partager vos créations et obtenir l'aide d'une communauté active et conviviale !
+## Rejoignez notre Communauté
 
-💬 Dites bonjour dans notre salon `#introduce-yourself` !
+Rejoignez-nous dans le [Discord Astro](https://astro.build/chat/) afin de partager et obtenir de l'aide de la part d'une communauté active et accueillante !
 
-💬 Demandez de l'aide à l'équipe de support dans notre salon `#support-threads` !
+💬 Dîtes bonjour dans notre channel `#introduce-yourself` !
 
-💬 Partagez ce sur quoi vous travaillez en ce moment dans notre salon `#showcase`
+💬 Posez une question à notre équipe d'aide dans notre channel `#support-threads` !
 
-## Apprendre Encore Plus
+💬 Partagez ce sur quoi vous avez travaillé dans notre channel `#showcase` !
 
-[Blog d'Astro](https://astro.build/blog/)
+## En savoir plus
 
-[Note de Mise à Jour d'Astro](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
+[Blog Astro](https://astro.build/blog/)
 
-[Guide de Migration d'Astro](/fr/migrate/)
+[Changelog Astro](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
 
```

Comment:
```suggestion

[Guide de Migration Astro](/fr/migrate/)

```

Reply:
Migrate page does not exist yet in French.

Reply:
I think we can reference them anyway, it automatically appends "(EN)" to the text of the anchor and redirects to the english version.
It was done [here](https://github.com/withastro/docs/blob/main/src/pages/fr/reference/configuration-reference.md?plain=1#L174 ) and [here](https://github.com/withastro/docs/blob/main/src/pages/fr/concepts/islands.md?plain=1#L64)

Reply:
Ok, done.